bftengine/src/bcstatetransfer/Messages.hpp

```diff
@@ -40,8 +40,12 @@ class MsgType {
 };
 
 struct BCStateTranBaseMsg {
-  BCStateTranBaseMsg(uint16_t type) : type(type) {}
+  BCStateTranBaseMsg(uint16_t type) : BCStateTranBaseMsg(type, false) {}
+  BCStateTranBaseMsg(uint16_t type, bool isIncomingMsg) : type(type), is_incoming_msg_(isIncomingMsg) {}
```

Comment:
please use single ctor, it doesn make sense here ot hold 2:
`BCStateTranBaseMsg(uint16_t type, bool isIncomingMsg=false)`